web/src/routes/SurveyPages/SurveyListPage.tsx

```diff
@@ -1,95 +1,95 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 
-import { AxiosError, AxiosResponse } from 'axios';
+import { useQuery } from '@tanstack/react-query';
+import { AxiosError } from 'axios';
 import styled from 'styled-components';
 
-import axios from '../../api/axios';
-import { requests } from '../../api/request';
+import { fetchSurveyList } from '../../api/fetchFunctions';
+import ErrorPage from '../../components/ErrorPage';
 import Header from '../../components/Header';
 import { SurveyPreviewModal } from '../../components/Modal';
 import Pagination from '../../components/Pagination';
 import SurveyListSkeleton from '../../components/Skeleton/SurveyListSkeleton';
 import SurveyListTable from '../../components/SurveyListTable';
 import { useTheme } from '../../hooks/useTheme';
-import { SurveyAbstractResponse, SurveyPageResponse } from '../../types/response/Survey';
+import { SurveyPageResponse } from '../../types/response/Survey';
 
 const Container = styled.div`
   width: 100vw;
   height: 100vh;
   background-color: ${(props) => props.theme.colors.container};
 `;
 
+const ListContainer = styled.div``;
+
 export default function SurveyListPage() {
   const [theme, toggleTheme] = useTheme();
-  const [isLoading, setIsLoading] = useState<boolean>(false);
-  const [abortController, setAbortController] = useState<AbortController>(new AbortController());
-  const [surveys, setSurveys] = useState<SurveyAbstractResponse[]>([]);
   const [page, setPage] = useState<number>(1);
-  const [totalPages, setTotalPages] = useState<number>(0);
   const [previewModalOpen, setPreviewModalOpen] = useState<boolean>(false);
   const [selectedSurveyIndex, setSelectedSurveyIndex] = useState<number>(0);
 
-  const fetchSurveyList = async (abortSignal: AbortSignal): Promise<void> => {
-    setIsLoading(true);
-    try {
-      const request: AxiosResponse<SurveyPageResponse> = await axios.get<SurveyPageResponse>(
-        `${requests.getSurveyPage}${page}`,
-        { signal: abortSignal }
-      );
+  // FIXME: cacheTime and staleTime is appropriate?
+  const { data, isLoading, isError, error } = useQuery<SurveyPageResponse>(['surveyPage', page], fetchSurveyList, {
+    cacheTime: 9000,
+    staleTime: 60000,
```

Comment:
`cacheTime` should be bigger than `staleTime`. And your `cacheTime` is way too tight and `staleTime` doesn't make sense since it's **Survey List Page** that needs to show fresh data from server.

Please modify it to

```typescript
cacheTime: 5 * 60 * 1000, // 5 minutes
staleTime: 20 * 1000, // 20 seconds
```